recipes/ninja/1.10/conanfile.py

```diff
@@ -0,0 +1,43 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class NinjaConan(ConanFile):
+    name = "ninja"
+    description = "Ninja is a small build system with a focus on speed"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ninja-build/ninja"
+    settings = "os_build", "arch_build"
```

Comment:
I think we can start using the new cross-building approach. WDYT @jgsogo

Reply:
Let's wait for 1.25 and see if we can implement all the use cases with the new approach. At that moment we can start recommending it and, probably, we will mark the old approach as deprecated (using `CONAN_V2_MODE`). 

Right now there are many recipes in ConanCenter with `os_build` and `arch_build`. We cannot force this one recipe to use a different approach without making it a _general recommendation_.